lib/recognizer_web/router.ex

```diff
@@ -29,8 +29,22 @@ defmodule RecognizerWeb.Router do
     plug Guardian.Plug.EnsureNotAuthenticated
   end
 
+  pipeline :bc do
+    plug :add_bc_to_session
+  end
+
+  defp add_bc_to_session(%{query_params: %{"bc" => "true"}} = conn, _opts) do
+    if Recognizer.BigCommerce.enabled?() do
+      put_session(conn, :bc, true)
+    end
+  end
```

Comment:
would be an abnormal case, but you might want `conn` in an `else` here for safety

Reply:
good point!